### ISSUE TITLE
nvda does not report table on new gmail page

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8808]
### Steps to reproduce:
Open the gmail page and log into your account
Use the modern default gmail preview page.
Search for an email using gmail search.
I believe this also works on the first page of the inbox with multiple emails.
after in the search box for emails in gmail press enter to perform the search and press t to go to the local message table
### Actual behavior:
In chrome 69 the nvda will find a table but not the table d
messages, you can press shift + h to go to the search result header.
press down arrow nvda announces the beginning of messages as a checkbox, and nvda does not announce that it is in a table, nvda ignores that the messages are organized as a table.
In firefox 62 the nvda even informs that it is a table, however it says that the table has 0 rows and 0 columns, and does not allow to navigate the table using the commands ctrl + alt + arrows.
### Expected behavior:
Nvda has to report that it is a table and must allow navigation in the table.
Maybe this started after updating gmail.
#### NVDA Installed/portable/running from source:
instaled
#### NVDA version:
alfa 16123
#### Windows version:
10 17134.320
#### Name and version of other software in use when reproducing the issue:
chrome 69 e firefox 62 64bits
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
[gmail.txt](https://github.com/nvaccess/nvda/files/2432180/gmail.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva this is solved in Firefox 66.0.2. At least it works for me with NVDA 2019.1. Can you confirm?
The chrome issue is still valid, but this could be also due to implementation aspects and how the browser exposes the table to NVDA. See i.e. the statement from @feerrenrut in #7852. I think there is no way to provide a consinstent heuristic to distinguish between different table structures and how they are exposed in different browsers. It will be up to the web developer to expose it correctly. But I might be wrong, so I will let it to more experienced developers to assess if this issue can be solved from NVDA's side or not. Thanks!

**Comment by fernando-jose-silva** [comment]
Thank you very much.
I confirmed, in firefox everything works fine, in chrome it's in trouble.

**Comment by Adriani90** [comment]
This works correctly now in Chrome 81 with NVDA alpha-20097,6a573cd9. Closing as works for me.
